pyproject.toml

```
@@ -1,15 +1,96 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "scikit-build-core",
+    "swig",
     "oldest-supported-numpy; python_version < '3.9'",
     "numpy>=2.0.0; python_version >= '3.9'",
 ]
-build-backend = "setuptools.build_meta"
+build-backend = "scikit_build_core.build"
 
+[project]
+name = "wasserstein"
+version = "1.1.0"
+description = "Python package wrapping C++ code for computing Wasserstein distances"
+readme = "README.md"
+license.file = "LICENSE"
+requires-python = ">= 3.7"
+authors = [
+    { name = "Patrick T. Komiske III", email = "pkomiske@mit.edu" },
+]
+keywords = [
+    "distance",
+    "earth mover's distance",
+    "emd",
+    "energy mover's distance",
+    "fastjet",
+    "fjcontrib",
+    "hidden geometry",
+    "infrared safety",
+    "irc-safe",
+    "metric",
+    "optimal transport",
+    "ot",
+    "particle physics",
+    "wasserstein",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Natural Language :: English",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Unix",
+    "Programming Language :: C++",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: 3.10",
```

```suggestion
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
```

Can add Python 3.11 as we test against that now.